react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { switch: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  render() {
+    const Switch = this.state.switch;
+    if (Switch) {
+      return (
+        <div className = "container">
+          <div onClick = {this.handleClick} className = "on">
+            <div className = "shifter-on"></div>
+          </div>
+          <p>On</p>
+        </div>
+      );
+    }
+    return (
+      <div className = "container">
+        <div onClick = {this.handleClick} className = "off">
+          <div className = "shifter-off"></div>
+        </div>
+        <p>Off</p>
+      </div>
+    );
```

Comment:
Instead of having two separate JSX structures it would be better to use an `if` statement to determine the class names then have a single `return` with the class names set dynamically.